go.sum

```diff
@@ -0,0 +1,20 @@
+github.com/yuin/goldmark v1.1.27/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
```

Comment:
or this

Reply:
wasn't sure about these, but I'll remove